### PR TITLE
committed

### PR DESCRIPTION
**Pull request by ophir88**

### CODE REVIEW

File: Class.h [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 @interface Class : NSObject
 
-@property (nonatomic) BOOL blah;
```
Comment (xorserer) [inline Class.h]: Comment 1

File: Class.h [hunk 2]
```diff
@@ -2,6 +2,6 @@
...
+@property (nonatomic) BOOL blahBla;
```
Comment (xorserer) [inline Class.h]: Another comment

File: Class.h (hunk 1 above)
Comment (xorserer) [inline Class.h]: Another line comment

**Comment by xorserer** [comment]
Round 1 done.

File: Class.h (hunk 1 above)
Comment (ophir88) [inline Class.h]: ok.

**Comment by ophir88** [comment]
round 1 done

**Pull request merged**